DuckDuckGo/ImageCacheDebugViewController.swift

```diff
@@ -87,6 +87,30 @@ class ImageCacheDebugViewController: UITableViewController {
         }
     }
 
+    override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+
+        let host: String?
+        switch indexPath.section {
+        case 0: host = bookmarksManager.favorite(atIndex: indexPath.row)?.url.host
+        case 1: host = bookmarksManager.bookmark(atIndex: indexPath.row)?.url.host
+        case 2: host = tabsModel.get(tabAt: indexPath.row).link?.url.host
```

Comment:
Have you considered mapping the sections to an enum? If we end up doing more with this, it could make moving things around etc much easier